packages/shared/lib/migration.ts

```diff
@@ -169,6 +170,15 @@ export const getMigrationData = (migrationSeed: string, initialAddressIndex = 0)
             reject({ snapshot: true })
             openSnapshotPopup()
         } else {
+            // Generate address using iotajs
+            const legacyAddresses: string[] = []
+            const binaryAddresses: string[] = []
+            for (let index = 0; index < 10; index++) {
+                const legacyAddress = generateAddress(migrationSeed, index, 2)
```

Comment:
```suggestion
                const legacyAddress = generateAddress(migrationSeed, index, ADDRESS_SECURITY_LEVEL)
```

Reply:
Also, based on what was shared in the slask channel, we might have to remove the last 9 characters as they are the checksum of the old address format, i dont know if we have to remove them, but just to be aware of this

Reply:
It comes with the correct format of 81 characters in length